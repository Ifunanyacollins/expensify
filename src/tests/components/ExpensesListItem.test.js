import React from 'react';
import { shallow } from 'enzyme';
import  ExpenseListItem  from '../../components/ExpensesListItem';
import { expenses } from '../fixture'

test("should render each expenses" , () => {
 const wrapper = shallow(<ExpenseListItem  {...expenses[0]}/>)
 expect(wrapper).toMatchSnapshot()
})