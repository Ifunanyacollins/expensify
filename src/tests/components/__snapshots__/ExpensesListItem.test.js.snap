// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render each expenses 1`] = `
<div>
  <p>
    Title of Payment = &gt;
    <Link
      replace={false}
      to="/edit/1234567"
    >
      I bought goat
    </Link>
  </p>
  <p>
     Amount = &gt; 
    200
  </p>
  <p>
    Time & Date = &gt;
    1538130411204
  </p>
  <p>
    Note = &gt; 
    i got molly i black sheep
  </p>
  <br />
</div>
`;
